aws/common/vpc.tf

```
@@ -150,6 +194,29 @@ resource "aws_route_table_association" "notification-canada-ca-private" {
   route_table_id = aws_route_table.notification-canada-ca-private_subnet.*.id[count.index]
 }
 
+resource "aws_route_table" "notification-canada-ca-private_subnet_new" {
+  count = 3
+
+  vpc_id = aws_vpc.notification-canada-ca.id
+
+  route {
+    cidr_block     = "0.0.0.0/0"
+    nat_gateway_id = aws_nat_gateway.notification-canada-ca-new.*.id[count.index]
+  }
+
+  tags = {
+    Name       = "Private Subnet Route Table ${count.index}"
```

Minor suggestion to add a value to keep them separate from the existing 3 private subnet route table names:
```suggestion
    Name       = "Private Subnet Route Table ${count.index + 3}"
```